### PR TITLE
snap: some doc comments fixes and additions

### PR DESCRIPTION
**Pull request by pedronis**
I did a quick review over snap package doc comments. This is the result with some fixes and additions.

### CODE REVIEW

File: snap/info.go [hunk 1]
```diff
@@ -689,6 +691,7 @@ func (app *AppInfo) SecurityTag() string {
 	return AppSecurityTag(app.Snap.Name(), app.Name)
 }
 
+// DesktopFile returns the path to the installed optional desktop file for the application.
```
Comment (bboozzoo) [inline snap/info.go]: `.. , optionally installed, application desktop file`?

File: snap/broken.go [hunk 2]
```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/snapcore/snapd/dirs"
 )
 
+// GuessAppsForBroken guesses what apps and services a broken snap has
+// on the system by searching for matches based on the snap name in
```
Comment (bboozzoo) [inline snap/broken.go]: I think it's `s/on/in/`, but better ask a native speaker, @chipaca?

File: snap/info.go (hunk 1 above)
Comment (pedronis) [reply snap/info.go]: the original file is optional but if present is installed. This formulation is quirky but the other doesn't convey that to me

File: snap/broken.go (hunk 2 above)
Comment (pedronis) [reply snap/broken.go]: I know it's usually 'on' disk, and slightly more often 'in' the filesystem than 'on' the filesystem.

For apps 'in' vs 'on' the system, I can find both (as in googling). 'on' seems more common. Not sure one is more correct than the other. They seem slightly different views on the situation.

Comment (chipaca) [reply snap/broken.go]: Both seem correct to me as well. I'd've gone with 'in' myself, but that's probably more about me than about what's being said.

File: snap/info.go (hunk 1 above)
Comment (chipaca) [reply snap/info.go]: How about
> ```
> DesktopFile returns the installed path to the optional desktop file for the application.
> ```
?

File: snap/info.go [hunk 3]
```diff
@@ -898,7 +901,7 @@ func ReadCurrentInfo(snapName string) (*Info, error) {
 	return ReadInfo(snapName, &SideInfo{Revision: revision})
 }
 
-// ReadInfoFromSnapFile reads the snap information from the given File
+// ReadInfoFromSnapFile reads the snap information from the given Container
```
Comment (chipaca) [inline snap/info.go]: . at the end, while you're doing this :-)

**Pull request merged**